src/helpers/Mail/templates/main.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-console */
+/* eslint-disable no-undef */
+$(document).ready(() => {
+  const socket = io('http://localhost:5000');
```

Comment:
This can be added from the .env file to make this action dynamic and easy to switch when we deploy